src/axios.js

```diff
@@ -0,0 +1,8 @@
+import axios from 'axios';
+
+export const api = axios.create({
+	baseURL: 'http://localhost:8080/api/',
+	headers: {
+		'content-type': 'application/json',

```

Comment:
It's better to store backend address in .env file. That way we have more flexibility when deploying the app.

Reply:
I've added `.env` and updated CORS header in backend (it's deployed to production).
Please update frontend code so that a smell without any occurrences is not visible (or is visible in another group i.e. "not found").